src/client/pages/DeleteAccountBlocked.tsx

```diff
@@ -18,18 +18,22 @@ export const DeleteAccountBlocked = ({ contentAccess = {} }: Props) => {
 		contentAccess.guardianWeeklySubscriber;
 
 	return (
-		<MainLayout pageHeader="Delete your Guardian account">
-			<MainBodyText>
-				We&apos;ve noticed you have an active membership, recurring contribution
-				or subscription with the Guardian, so you won&apos;t be able to
-				automatically delete your account online. Please follow the instructions
-				below first and try again if you would still like delete your account.
-			</MainBodyText>
-
+		<MinimalLayout
+			pageHeader="Delete your Guardian account"
+			leadText={
+				<MainBodyText>
+					We&apos;ve noticed you have an active membership, recurring
+					contribution or subscription with the Guardian, so you won&apos;t be
+					able to automatically delete your account online. Please follow the
+					instructions below first and try again if you would still like delete
+					your account.
+				</MainBodyText>
+			}
```

Comment:
I think this should be separated into two paragraphs, as it becomes unwieldy to read?

Something close to this?

Again, it was like this previously, but could be an ideal time to change it.

```suggestion
			leadText={
			  </>
				<MainBodyText>
					We&apos;ve noticed you have an active membership, recurring
					contribution or subscription with the Guardian, so you won&apos;t be
					able to automatically delete your account online.
				</MainBodyText>
				<MainBodyText>
				Please follow the
					instructions below first and try again if you would still like delete
					your account.
					</MainBodyText>
				</>
			}
```